setup.py

```diff
@@ -109,7 +109,7 @@ def long_description():
         "zulip>=0.8.2,<0.9.0",  # Next release, 0.9.0, requires Python 3.9
         "urwid_readline>=0.15.1",
         "beautifulsoup4>=4.13.4",
-        "lxml==4.9.4",
```

Comment:
I've been busy outside of ZT recently, but I believe this was pinned so as to avoid errors with PyPy 3.7, which I believe you're seeing here.

As per comments elsewhere - likely in the channel(s) on chat.zulip.org as well as on GitHub - the idea some time ago was to get a release out for 3.7 and then drop support. I don't recall if one can set different package version dependencies per python version, which would work as a short-term workaround for this. Failing that, I'd be open to moving to 3.9+, since 3.8 is also past end of life now.